### ISSUE TITLE
[DX] FEATURE REQUEST: APX-like CLI utility

### ISSUE THREAD

**Issue by Malix-off** [ublue-os/bluefin#1498]
### Describe the package

[APX](https://github.com/Vanilla-OS/apx) is the package manager of [VanillaOS](https://vanillaos.org/)
It has been developed by @89luca89, a [co-founder of VanillaOS](https://vanillaos.org/team) and [the founder of Distrobox](https://github.com/89luca89/distrobox/graphs/contributors)

[Here is a blog post about what it is, how it works, and what is its added value](https://vanillaos.org/blog/article/2023-01-28/apx-an-unconventional-package-manager)

What it does is that it centralizes and abstracts the creation of distroboxes and package install through podman

i.e., the single `apx --dnf install steam` command install steams through dnf in a fedora distrobox managed by podman

Its new version (v2) [has been released just now](https://github.com/Vanilla-OS/apx/releases), and [here is a blog post about it](https://vanillaos.org/blog/article/2024-07-11/discover-apx-v2-the-new-essential-tool-for-developers-and-creators-on-vanilla-os-orchid)
Basically, it switched from being a package manager for various package formats which were installed into containers; to a container manager in which you can create and manage different containers easily and install packages from different distros

### Image

DX variants

**Comment by castrojo** [comment]
This is pretty cool but we're going in a different direction, our pattern is devcontainers and flatpaks.

**Comment by Malix-off** [comment]
I was mistaken, my bad

APX is not only for DX, but for all images

It simplify distrobox use-cases

**Comment by Malix-off** [comment]
Can we consider a re-opening?

**Comment by m2Giles** [comment]
https://github.com/Vanilla-OS/apx?tab=readme-ov-file#installing-apx-in-a-custom-destination
